e2e/tests/caseManagementOrder_test.js

```diff
@@ -104,7 +104,79 @@ Scenario('Other parties can see the draft CMO document when it is marked for par
   }
 });
 
-// Scenario('Judge actions CMO', async (I, caseViewPage, draftCaseManagementOrderEventPage, actionCaseManagementOrderEventPage))
+Scenario('Local Authority sends draft to Judge who approves CMO', async (I, caseViewPage, draftCaseManagementOrderEventPage, actionCaseManagementOrderEventPage) => {
+  // LA sends to judge
+  await caseViewPage.goToNewActions(config.applicationActions.draftCaseManagementOrder);
+  await skipToReview(I, ids);
+  draftCaseManagementOrderEventPage.markToBeSentToJudge();
+  await I.completeEvent('Submit');
+  I.dontSee('Draft orders', '.tabs .tabs-list');
+
+  // Login as Judge
+  await I.signOut();
+  await I.signIn(config.judiciaryEmail, config.judiciaryPassword);
+  await I.navigateToCaseDetails(caseId);
+  assertCanSeeDraftCMO(I, caseViewPage, draftCaseManagementOrderEventPage.staticFields.statusRadioGroup.sendToJudge);
+
+  // Approve CMO
+  await caseViewPage.goToNewActions(config.applicationActions.actionCaseManagementOrder);
+  await skipToReview(I, ids.slice(1, ids.length - 1));
```

Comment:
Ideally the tests should be readable from functionality perspective - I don't like it, it's not obvious what's happening.

Helper method maybe, skipToActionPage? could be enough.

Reply:
I've extracted this into a helper method skipToSchedule since at this point, this is where we'd like to go to